packages/react-widgets/src/widgets/FormulaWidget.js

```diff
@@ -19,6 +19,7 @@ import { columnAggregationOn } from './utils/propTypesFns';
  * @param  {AggregationTypes} props.operation - Operation to apply to the operationColumn. Must be one of those defined in `AggregationTypes` object.
  * @param  {Function} [props.formatter] - Function to format each value returned.
  * @param  {boolean} [props.animation] - Enable/disable widget animations on data updates. Enabled by default.
+ * @param  {boolean} [props.global] - 
```

Comment:
Missing doc. Better fill it now or that's the kind of thing that stays like that forever.

Reply:
Yes, I agree with you. I'm sorry that you are reviewing a very dirty code 😆 My idea here was the validation of the general approach from Victor, so I made this very quickly. I'll add this in the following commit